curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
This will need a guaranteed() method similar to node deletion. Use the FailedDeleteManager as a model (maybe extend it?).